CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * Fixed `RuntimeError` when using `compas_rhino.unload_modules` in CPython`. 
+* Fixed `scaled` in `Box`.
```

Comment:
I assumed the original issue is about `Geometry.scaled` not working quite right for `Box` (I think it's using `transfome(Scale(...)))`.. did it just move `Box.frame` or something? perhaps a better description of what used to happen here?

Reply:
Updated the changelog!